apiserver/facades/client/machinemanager/machinemanager.go

```diff
@@ -353,3 +379,15 @@ func (mm *MachineManagerAPIV4) updateOneMachineSeries(arg params.UpdateSeriesArg
 	}
 	return machine.UpdateMachineSeries(arg.Series, arg.Force)
 }
+
+func (mm *MachineManagerAPIV5) createUpgradeSeriesPrepareLock(arg params.UpdateSeriesArg) error {
+	machineTag, err := names.ParseMachineTag(arg.Entity.Tag)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	machine, err := mm.st.Machine(machineTag.Id())
+	if err != nil {
+		return errors.Trace(err)
+	}
+	return machine.CreateUpgradeSeriesPrepareLock()
```

Comment:
machine.CreateUpgradeSeriesPrepareLock() does seem correct.  we can add other machine.<func> calls before or after as they are created.

Reply:
@hmlanigan  Ah, I see what you mean. The name "doesn't" seem correct since it is not really "locking" the machine per se but only prevents certain commands from being run. Do you have any suggestions.